Games/types/Mafia/roles/cards/Polariser.js

```diff
@@ -8,7 +8,7 @@ module.exports = class Polariser extends Card {
     this.meetings = {
       Polarise: {
         states: ["Night"],
-        flags: ["voting", "multi", "group", "speech"],
+        flags: ["group", "speech", "voting", "multiActor"],
```

Comment:
did you want it to be both multi and multiactor?

Reply:
no, just multiactor